Dockerfile

```
@@ -6,13 +6,18 @@ USER root
         python3-pysvn \
         python3-pip \
         python3-dev \
         libpq-dev \
         build-essential \
         libmagic1 && \
-    python3 -m pip install --upgrade pip && \
+    python3 -m venv /local/venv --system-site-packages
+
+ENV PATH="/local/venv/bin:$PATH"
+
+RUN python3 -m pip install --upgrade pip && \
```

Hadolint (2.14.0)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)